src/api.ts

```diff
@@ -0,0 +1,11 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const getUsers = (userId: number) => {
```

Comment:
You're fetching only one user, why plural?
```suggestion
export const getUser = (userId: number) => {
```

Reply:
Done